### DISCUSSION TITLE
Ongoing pinned topic of themer working alongside other libraries

### DISCUSSION THREAD

**Discussion by TriStarGod** [RyanClementsHax/tailwindcss-themer#79] [Ideas]
Would be great to have a community contributed list of other tailwind plugin libraries (daisy ui, skeleton ui, etc) that work alongside themer.

**Comment by RyanClementsHax** [comment]
I'm definitely up for it!

I'm not sure what other people use, as I currently don't have a way of taking a pulse on how people use this plugin, but if others want to share compatibility I'd be happy to make a list.
